figo/Config.php

```diff
@@ -29,8 +29,8 @@
  */
 class Config {
 
-    /** @var string figo Connect server hostname */
-    public static $API_ENDPOINT = "api.figo.me";
+    /** @var string figo Connect server address. This should be the full base url of the API */
+    public static $API_ENDPOINT = "https://api.figo.me/v3";
```

Comment:
Do we still support the legacy way of setting the `API_ENDPOINT` (just `api.figo.me`)? Do you handle trailing slashes?

Reply:
yes we support both ways now